tests/foreman/api/test_syncplan.py

```diff
@@ -835,11 +838,11 @@ def test_positive_synchronize_custom_products_future_sync_date(self):
             with self.assertRaises(AssertionError):
                 self.validate_task_status(repo.id)
         # Create and Associate sync plan with products
-        if bz_bug_is_open(1695733):
-            self.logger.info('Need to set seconds to zero because BZ#1695733')
-            sync_date = datetime.utcnow().replace(second=0) + timedelta(seconds=delay)
-        else:
-            sync_date = datetime.utcnow() + timedelta(seconds=delay),
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1695733 still an open bug, why did else condition was removed here? if it valid then The same case also should be updated in test test_positive_synchronize_custom_product_future_sync_date

Reply:
yes, line 784 & 785 needs to be copied to 845